packages/dx-react-scheduler/src/types/editing/appointment-tooltip.types.ts

```diff
@@ -28,19 +30,41 @@ export namespace AppointmentTooltip {
     contentComponent: React.ComponentType<AppointmentTooltip.ContentProps>;
     /** A component that renders a command button. */
     commandButtonComponent: React.ComponentType<AppointmentTooltip.CommandButtonProps>;
+    /** A component that renders an icon for recurring appointments inside the AppointmentTooltip. */
+    recurringIconComponent: React.ComponentType<object>;
   }
   /** Describes properties passed to a component that renders the tooltip header. */
   export interface HeaderProps {
     /** The appointment’s displayed metadata. */
     appointmentData?: AppointmentModel;
-    /** A React node used to render the tooltip header. */
+    /** Specifies the Open button’s visibility. */
+    showOpenButton: boolean;
+    /** Specifies the Close button’s visibility. */
+    showCloseButton: boolean;
+    /** Specifies the Delete button’s visibility. */
+    showDeleteButton: boolean;
+    /** A command button’s identifier list. */
+    commandButtonIds: Array<string>;
+    /** An event raised when the Open button is clicked. The event handler should open the appointment form. */
+    onOpenButtonClick?: () => void;
+    /** An event raised when the Delete button is clicked. The event handler should delete an appointment. */
+    onDeleteButtonClick?: () => void;
+    /** An event that hides the tooltip. */
+    onHide?: () => void;
+    /** A component that renders a command button. */
+    commandButtonComponent: React.ComponentType<AppointmentTooltip.CommandButtonProps>;
+    /** A React node used to render the additional part of the tooltip header. */
```

Comment:
There's only one additional part? If there can be different parts, use "an additional part".